src/Restore.php

```diff
@@ -890,4 +891,33 @@ public function restoreNotifications(): void
             ));
         }
     }
+
+    private function checkTableRestorable(array $tableInfo): void
+    {
+        // check if primage key has nullable column
+        if (($tableInfo['isTyped'] ?? false) === true) {
+            foreach ($tableInfo['columnMetadata'] as $columnName => $columnMetadata) {
+                if (!in_array($columnName, $tableInfo['primaryKey'], true)) {
+                    continue;
+                }
+                $columnMetadata = array_filter(
+                    $columnMetadata,
+                    fn($v) => $v['provider'] === 'storage',
+                );
+
+                $columnMetadatum = array_combine(
```

Comment:
Metadatum? 😄

Reply:
přečti si naší plodnou diskuzi na - https://keboolaglobal.slack.com/archives/C055HSMKX51/p1721215406702399?thread_ts=1721212949.196679&cid=C055HSMKX51
😃 😃 😃

Reply:
tak jsem to teda přejmenoval no :D